modules/dnn/src/model.cpp

```diff
@@ -1321,6 +1322,22 @@ struct TextDetectionModel_DB_Impl : public TextDetectionModel_Impl
     void setMaxCandidates(int maxCandidates_) { maxCandidates = maxCandidates_; }
     int getMaxCandidates() const { return maxCandidates; }
 
+    void setInputStd(const Scalar& std_) { std = std_; }
+
+    void processFrame(InputArray frame, OutputArrayOfArrays outs)
+    {
+        CV_TRACE_FUNCTION();
+        if (size.empty())
+            CV_Error(Error::StsBadSize, "Input size not specified");
+
+        blob = blobFromImage(frame, scale, size, mean, swapRB, crop);
+
+        if (!std.isReal())
+            blob /= std;
```

Comment:
There is some kind of mess between `scale` and `std`.

---

Right fix should include:
- overload `blobFromImage()` API to support `Scalar scale`
- overload `Model::setInputScale()` with `Scalar` input.

Otherwise we introduce mess.

Reply:
Hi @alalek, thanks for the code review.
How about overloading the` blobFromImage()` API to support `Scalar scale` and `Scalar std`?
For now, initialization of many models requires not only the **mean** but also the **standard deviation**.